src/distribution/cauchy.rs

```diff
@@ -0,0 +1,238 @@
+use distribution;
+use source::Source;
+
+/// A Cauchy distribution.
+///
+/// A Cauchy distribution (aka Lorentz or Cauchy–Lorentz distribution) is a continuous
+/// probability distribution with a location parameter `loc`, a scale parameter `gamma > 0`,
+/// and the following probability density function:
+///
+/// `p(x) = const / (1 + ((x - loc) / gamma)^2)`.
+///
+/// A Cauchy distribution is long tailed and has no well-defined mean or variance. It is
+/// unimodal with its mode at `loc`, around which it is symmetric. The ratio of two
+/// independent Gaussian distributed random variables is Cauchy distributed.
+///
+/// See [Wikipedia article on Cauchy
+/// distribution](https://en.wikipedia.org/wiki/Cauchy_distribution).
+#[derive(Clone, Copy, Debug)]
+pub struct Cauchy {
+    loc: f64,
```

Comment:
It is a bit inconsistent with respect to the rest of the crate. In all other cases, it is mathematical notation that is used, not names. For instance, it is called `mu` in `Gaussian`, even though it is also the location of the distribution, and it is called `sigma`, even though it is the scale.

So if we take the Wikipedia page as a reference, it should be `x_0`.